--- node_modules/nyc/index.js~	2021-11-15 22:53:17.000000000 +0100
+++ node_modules/nyc/index.js	2021-11-15 22:53:23.000000000 +0100
@@ -501,11 +501,6 @@
       var coverage = summary[key].pct
       if (coverage < thresholds[key]) {
         process.exitCode = 1
-        if (file) {
-          console.error('ERROR: Coverage for ' + key + ' (' + coverage + '%) does not meet threshold (' + thresholds[key] + '%) for ' + file)
-        } else {
-          console.error('ERROR: Coverage for ' + key + ' (' + coverage + '%) does not meet global threshold (' + thresholds[key] + '%)')
-        }
       }
     })
   }
